test/e2e/specs/participants-spec.js

```diff
@@ -0,0 +1,40 @@
+describe('Participants', () => {
+  beforeEach(() => {
+    cy.task('createUser', {
+      email: 'masa',
+      password: 'Salasana123',
+    })
+    cy.task('loadFixtures', 'participants-and-deps.json')
+    cy.login('masa', 'Salasana123')
+    cy.contains(/Leiriläiset/i).click()
+  })
+
+  // REKI shouldn't show results before searching to prevent users from accidentally
+  // seeing personally identifiable information
+  it('are not shown in the list initially', () => {
```

Comment:
this is good, should we also have one that tests clearing the search, to make sure that the list resets to zero

Reply:
Makes sense, I'll add that.